MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroiddemo/analytics/FirstTimeRunChecker.java

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.mapboxandroiddemo.analytics;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.mapbox.mapboxandroiddemo.BuildConfig;
+
+import static android.content.Context.MODE_PRIVATE;
+
+/**
+ * This class checks whether the app is being opened for the first time or not and adjusts shared preferences
+ */
+
+public class FirstTimeRunChecker {
```

Comment:
Another duplicate? You might want to share this with a shared module, for example.